engine/src/button.cpp

```diff
@@ -1382,7 +1365,6 @@ Boolean MCButton::mup(uint2 which, bool p_release)
                     // Check whether the click was inside the menu (the rect
                     // that we need to check is the rect of the stack containing
                     // the menu).
-                    MCRectangle t_rect = t_menu->getstack()->getrect();
                     t_outside = !MCU_point_in_rect(rect, mx, my);
```

Comment:
This looks like it might be a bug - I presume the test is meant to be `!MCU_point_in_rect(t_rect, mx, my);`, otherwise it's testing whether the point is outside the button rect...